appcues/src/main/java/com/appcues/ui/AppcuesViewModel.kt

```diff
@@ -141,6 +141,8 @@ internal class AppcuesViewModel(
     }
 
     fun onPause() {
+        if (uiState.value is Dismissing) return
```

Comment:
another little thing that was hit in UI tests it seemed in some race condition - even though the UI was dismissing, this viewModelScope below still let the Pause action through.  This seems rare, and I don't know how it happened - but we can guard against that pretty easily.